src/lib/game/world/handler.js

```diff
@@ -13,6 +13,17 @@ class WorldHandler extends EventEmitter {
     this.scene = new THREE.Scene();
     this.map = null;
 
+    this.prevCameraRotation = null;
+
+    // TODO: Use this handler for all entities, not just the player
+    this.player.on('change:model', (oldModel, newModel) => {
+      if (oldModel) {
+        this.scene.remove(oldModel);
+      }
+
+      this.scene.add(newModel);
+    });
```

Comment:
Ah, woops! This probably was a mistaken inclusion when I resolved merge conflicts from upstream.

I should remove everything from line 18 to line 25 above?